src/components/Certify/Certify.vue

```diff
@@ -1,80 +1,78 @@
 <template>
-  <v-card flat id="AR-step-4-container" class="pr-5">
-    <v-form id="certify-form" ref="certifyForm" lazy-validation class="mt-4" v-on:submit.prevent>
-      <v-container>
-        <v-row class="pl-2" no-gutters>
-          <v-col :cols="firstColumn">
-            <label :class="{'error-text': invalidSection}"><strong>Legal Name</strong></label>
-          </v-col>
-          <v-col :cols="secondColumn" class="px-0">
-            <div class="value certified-by">
-              <v-text-field
-                filled
-                persistent-hint
-                id="certified-by-textfield"
-                label="Legal name of authorized person"
-                :value="certifiedBy"
-                :rules="[(v) => !!v || 'A person\'s legal name is required.']"
-                :disabled="disableEdit"
-                @input="emitCertifiedBy($event)"
-              />
-            </div>
-          </v-col>
-        </v-row>
-        <v-row no-gutters class="pl-1">
-          <v-col :cols="firstColumn" />
-          <v-col :cols="secondColumn">
-            <v-checkbox
-              hide-details
-              :value="isCertified"
-              @change="emitIsCertified($event)"
-              id="isCertified-checkbox"
-            >
-              <template slot="label">
-                <div class="certify-stmt" :class="{'error-text': invalidSection && !isCertified}" v-if="isStaff">
-                  <strong>{{ trimmedCertifiedBy || "[Legal Name]" }}</strong>
-                  certifies that they have relevant knowledge of the
-                  {{ entityDisplay || "association" }} and is authorized to
-                  make this filing.
-                </div>
-                <div class="certify-stmt" :class="{'error-text': invalidSection && !isCertified}" v-else>
-                  I,
-                  <strong>{{ trimmedCertifiedBy || "[Legal Name]" }}</strong>
-                  , certify that I have relevant knowledge of the
-                  {{ entityDisplay || "association" }} and I am authorized to
-                  make this filing.
-                </div>
-              </template>
-            </v-checkbox>
-            <ul class="certify-statements mt-4">
-              <li v-for="(statement, index) in statements" :key="`statement-${index}`" class="pt-2">
-                {{ statement }}
+  <v-card flat id="AR-step-4-container" class="py-8 px-6">
+    <v-form ref="certifyForm" lazy-validation v-on:submit.prevent>
+      <v-row no-gutters>
+        <v-col cols="12" :sm="firstColumn" class="pr-4 pb-4">
```

Comment:
`cols="12"` means this column stacks (becomes full-width) below size sm.
`pr-4` is used to provide a small gutter so the column text doesn't bump into the next column.
`pb-4`is used to provide some whitespace between stacked columns.